packages/amplication-design-system/src/components/CircleBadge/CircleBadge.scss

```diff
@@ -18,3 +15,62 @@
   font-family: var(--primary-font-family);
   background-color: var(--secondary-turquoise);
 }
+
+.xxsmall {
+  width: 16px;
```

Comment:
please use params for all the values in the begining of this file

Reply:
they are not existed. I can create them' but it's specific to circle

Reply:
because they are specific, I asked to create SCSS param at the beginning of the file 
see this for example:
packages/amplication-client/src/Application/ApplicationHome.scss